### PR TITLE
created privacy policy, terms and conditions, and confirm credentials pages   

### PR DESCRIPTION
**Pull request by AlexandraVoloshyna**
## Describe your changes

- created privacy policy and terms and conditions pages
- added header, footer, main heading components
- created confirm credentials form/page
## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://olga-team.atlassian.net/browse/TP-59)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [ ] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit f

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

privacy_policy1
terms_and_condions1
terms_and_conditions2
confirm_credentials

### CODE REVIEW

File: src/components/confirmCredentialsForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useTranslation } from 'react-i18next';
+import { useUpdateMutation } from 'redux/authApiSlice';
+import { Inputs, Props } from './types';
+import { ButtonVariant } from '../button/types';
```
Comment (MishaFomenko) [inline src/components/confirmCredentialsForm/index.tsx]: we need absolute imports here

File: src/components/confirmCredentialsForm/confirmCredentialsFormHook.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+import { useTranslation } from 'react-i18next';
+
+export const useConfirmCredentialsSchema = () => {
+  const { t } = useTranslation();
+  const confirmSchema = yup
```
Comment (lyudad) [inline src/components/confirmCredentialsForm/confirmCredentialsFormHook.tsx:6]: empty line above

File: src/components/confirmCredentialsForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    formState: { errors },
+    reset,
+  } = useForm<Inputs>({
+    defaultValues: {
+      email: email_value,
+      full_name: full_name,
+    },
+    resolver: yupResolver(confirmSchema),
+  });
+  const onSubmit: SubmitHandler<Inputs> = async data => {
```
Comment (lyudad) [inline src/components/confirmCredentialsForm/index.tsx:50]: please check the order of the functions ad hooks, you have wrong

File: src/components/footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                </FooterMenuItem>
+              </FooterMenuList>
+            </FooterMenu>
+          </FooterRight>
+        </FooterInner>
+        <Credentials>© 2023 Black Cycle</Credentials>
+      </Container>
+    </StyledFooter>
+  );
+};
```
Comment (lyudad) [inline src/components/footer/index.tsx:111]: 1.3.3. PR have checklist, pleae check it before send things to review

Comment (AlexandraVoloshyna) [reply src/components/footer/index.tsx:111]: fixed in [91dd88a](https://github.com/ZenBit-Tech/faq_team_fe/pull/8/commits/91dd88a27d00755183ac4283d923f605ff2c23df)

File: src/components/confirmCredentialsForm/index.tsx (hunk 3 above)
Comment (AlexandraVoloshyna) [reply src/components/confirmCredentialsForm/index.tsx:50]: fixed in [5309628](https://github.com/ZenBit-Tech/faq_team_fe/pull/8/commits/53096286bb449eb7af490481ea7d5d267d756c46)

File: src/components/confirmCredentialsForm/confirmCredentialsFormHook.tsx (hunk 2 above)
Comment (AlexandraVoloshyna) [reply src/components/confirmCredentialsForm/confirmCredentialsFormHook.tsx:6]: fixed in [5309628](https://github.com/ZenBit-Tech/faq_team_fe/pull/8/commits/53096286bb449eb7af490481ea7d5d267d756c46)

Comment (lyudad) [reply src/components/confirmCredentialsForm/confirmCredentialsFormHook.tsx:6]: I still see the same here

File: src/components/footer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  FooterLogo,
+  FooterMenu,
+  FooterMenuItem,
+  FooterSocials,
+  FooterText,
+  FooterTitle,
+  FooterMenuList,
+  FooterIcon,
+  Credentials,
+} from './styles';
```
Comment (eugeneiv1) [inline src/components/footer/index.tsx:21]: path

File: src/components/confirmCredentialsForm/index.tsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+export const ConfirmCredentialsForm = ({
+  email_value,
+  full_name,
+  id,
+}: Props) => {
+  const { t } = useTranslation();
+  const [isSet, setIsSet] = useState(false);
+  const [update, { isLoading }] = useUpdateMutation();
+  const confirmSchema = useConfirmCredentialsSchema();
+  const {
```
Comment (lyudad) [inline src/components/confirmCredentialsForm/index.tsx:29]: add empty line above

File: src/components/confirmCredentialsForm/index.tsx [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    formState: { errors },
+    reset,
+  } = useForm<Inputs>({
+    defaultValues: {
+      email: email_value,
+      full_name: full_name,
+    },
+    resolver: yupResolver(confirmSchema),
+  });
+  useEffect(() => {
```
Comment (lyudad) [inline src/components/confirmCredentialsForm/index.tsx:43]: add empty line above

File: src/components/confirmCredentialsForm/index.tsx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+          message: errMsg,
+        });
+      } else if (isErrorWithMessage(err)) {
+        setError('root', {
+          message: err.message,
+        });
+      }
+    }
+  };
+  return (
```
Comment (lyudad) [inline src/components/confirmCredentialsForm/index.tsx:68]: empty line above

File: src/components/confirmCredentialsForm/types.ts [hunk 9]
```diff
@@ -0,0 +1,9 @@
+export type Inputs = {
+  email: string;
+  full_name: string;
+};
+export type Props = {
```
Comment (lyudad) [inline src/components/confirmCredentialsForm/types.ts:6]: empty line above

File: src/components/header/index.tsx [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+  NavLink,
+  StyledHeader,
+} from './styles';
+import { Container } from 'components/section/styles';
+import { useAppSelector } from 'redux/hooks';
+import { Button } from 'components/button';
+import { ButtonVariant } from 'components/button/types';
+import { useTranslation } from 'react-i18next';
+
+const productsNum = 2;
```
Comment (lyudad) [inline src/components/header/index.tsx:26]: add comment //TODO delete when functionality will be implemented

File: src/components/header/index.tsx [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+  MenuNavItem,
+  Nav,
+  NavLink,
+  StyledHeader,
+} from './styles';
+import { Container } from 'components/section/styles';
+import { useAppSelector } from 'redux/hooks';
+import { Button } from 'components/button';
+import { ButtonVariant } from 'components/button/types';
+import { useTranslation } from 'react-i18next';
```
Comment (lyudad) [inline src/components/header/index.tsx]: imports order

File: src/redux/authApiSlice.ts [hunk 12]
```diff
@@ -2,15 +2,18 @@ import { apiSlice } from 'redux/apiSlice';
...
   ResponseLogin,
   ResponseRegistration,
   RequestNewPass,
   ResponseVerifyOtp,
   RequestVerifyOtp,
 } from 'redux/types';
 import { paths } from 'const/paths';
-
-const AUTH_URL = '/auth';
+const AUTH_URL = 'Authorization';
```
Comment (lyudad) [inline src/redux/authApiSlice.ts]: empty line above

Comment (lyudad) [reply src/redux/authApiSlice.ts]: update to last path (small letter)

File: src/redux/types/index.ts [hunk 13]
```diff
@@ -34,6 +34,29 @@ export type AuthState = {
...
+  address?: string;
+  address_2?: string;
+  country?: string;
+  city?: string;
+  cloth_size?: string;
+  jeans_size?: string;
+  shoes_size?: number;
+};
+
+export type ResponseGetUser = {
```
Comment (lyudad) [inline src/redux/types/index.ts:52]: double check that `?` needed in all fields

**Pull request merged**